script/test_source_data_format.py

```diff
@@ -0,0 +1,34 @@
+import unittest;
```

Comment:
Semicolon is unnecessary.
I have fixed it in the master branch.

Reply:
Thx